politeiawww/api/v1/v1.go

```diff
@@ -381,6 +381,9 @@ type ProposalRecord struct {
 	NumComments         uint        `json:"numcomments"`                   // Number of comments on the proposal
 	Version             string      `json:"version"`                       // Record version
 	StatusChangeMessage string      `json:"statuschangemessage,omitempty"` // Message associated to the status change
+	PublishedAt         int64       `json:"publishedat,omitempty"`         // The timestamp of when the proposal has been published
```

Comment:
uint32 is more than enough for timestamp values. This way you reduce memory usage.
it should cover until 2106 (I guess you wont be the one refactoring this code =D)

consider making such refactor to improve memory usage

Reply:
UNIX timestamps are `int64`.

https://golang.org/pkg/time/#Time.Unix

Reply:
I am talking about how it is persisted in memory, not exactly about how other methods expect it. Considering it is an in memory database, it is super important to keep this in mind